Framework/Geometry/test/StructuredDetectorTest.h

```diff
@@ -151,18 +163,18 @@ class StructuredDetectorTest : public CxxTest::TestSuite {
     std::vector<double> y{0, 0, 0, 1, 1, 1};
 
     // Initialize with these parameters
-    TS_ASSERT_THROWS(det->initialize(2, 2, x, y, 0, true, 2, 1),
+    TS_ASSERT_THROWS(det->initialize(2, 2, x, y, "z", 0, true, 2, 1),
```

Comment:
Good. For throwing exceptions, I tend to use the `TSM_ASSERT_THROWS` so that I can include a failure message explaining why I expect this to throw.